src/store.js

```diff
@@ -0,0 +1,7 @@
+import { createStore } from 'redux';
+import reducer from './reducers';
```

Comment:
Looks like the pathing is off. Also your `faceReducers` doesn't have a default export.

Reply:
thanks Jose!